include/counter/device_info.h

```diff
@@ -0,0 +1,38 @@
+#ifndef DEVICEINFO_H
+#define DEVICEINFO_H
+
+#include <iostream>
+
+using namespace std;
+
+/*! \brief class DeviceInfo, holds device specific information
+ */
+class DeviceInfo {
+ public:
+  std::string machine = "";
+  std::string arch = "";
+  std::string osname = "";
+  std::string name = "";
+  std::string dist = "";
+  std::string uuid = "";
+
+  std::string get_info() const
+  {
+    std::string str = " \"os distribution\":";
```

Comment:
why can't we std::format?

Reply:
std::format availble in C++20 (Refered Link :https://en.cppreference.com/w/cpp/utility/format/format).
Are we using C++20x in DLR code base ?

Reply:
We have added string formating functionality as per available function in C++ 11x.